ImageFinder/View/ImageDetailViewController.swift

```diff
@@ -49,39 +49,40 @@ class ImageDetailViewController: UIViewController {
   
   override func viewDidLoad() {
     super.viewDidLoad()
-
     setView()
     layout()
   }
-  
+
+  // MARK: - Layout
+
   func setView() {
     navigationController?.isNavigationBarHidden = false
-
-    // imageView.image = UIImage(name: presentingViewController.getSelectedImage!)
     view.addSubview(scrollView)
-    imageView.image = UIImage(named: imageUrl ?? "")
-    scrollView.addSubview(imageView)
-    view.addSubview(imageSourceLabel)
-    view.addSubview(dateLabel)
+    scrollView.addSubview(contentsView)
+    contentsView.addSubview(imageView)
+    contentsView.addSubview(imageSourceLabel)
+    contentsView.addSubview(dateLabel)
   }
   
   func layout() {
-    scrollView.heightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.heightAnchor).isActive = true
-    scrollView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
-    scrollView.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor).isActive = true
-    scrollView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
-    scrollView.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor).isActive = true
+    scrollView.widthAnchor.constraint(equalTo: view.widthAnchor).isActive = true
+    scrollView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+    scrollView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
+    scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
```

Comment:
```suggestion
NSLayoutConstraint.activate([
    scrollView.widthAnchor.constraint(equalTo: view.widthAnchor),
    scrollView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
    scrollView.topAnchor.constraint(equalTo: view.topAnchor),
    scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
])
```

이런식으로 작성한다면 .isActive = true 를 작성하지 않아서 더 편리합니다.
저희는 위와 같은 방식으로 가시죠
해당 프로젝트에 적용 시키지는 않으셔도 됩니다. 
다음 프로젝트부터 사용하는걸로 하겠습니다.